packages/luxon/src/luxon-utils.ts

```diff
@@ -6,17 +6,17 @@ export default class LuxonUtils implements IUtils<DateTime> {
 
   public yearFormat = "yyyy";
 
-  public yearMonthFormat = "MMMM yyyy";
+  public yearMonthFormat = "LLLL yyyy";
 
   public dateTime12hFormat = "ff";
 
-  public dateTime24hFormat = "LLLL dd T";
+  public dateTime24hFormat = "MMMM dd T";
 
-  public time12hFormat = "hh:mm a";
+  public time12hFormat = "t";
```

Comment:
IMHO, 12/24 time formats should follow the same rules, so both are localizable formats. Therefore they should be specified or
```js
public time12hFormat = "t";
public time24hFormat = "T";
```
or
```js
public time12hFormat = "hh:mm a";
public time24hFormat = "HH:mm";
```

Reply:
I am wondering that `t` and `T` for some locales can be super long.

Reply:
I doubt, the luxon implementation is restricted by Intl API, where I cannot imagine a verbose format for time, due to `{ hour: n, minute: d2 }` etc... Do you know a locale where it could be long?